### PR TITLE
update styling on home page and link from home page to create post

### PR DESCRIPTION
**Pull request by TheunisK**

### CODE REVIEW

File: src/client/components/homeComponents/HomePosts.jsx [hunk 1]
```diff
@@ -46,6 +48,10 @@ const HomePosts = (props) => {
     getPosts();
   }, []);
 
+  const handleClick = () => {
+    console.log("Theunis");
+  }
+
```
Comment (Can-Tina) [inline src/client/components/homeComponents/HomePosts.jsx:54]: ?

File: src/client/components/homeComponents/HomePosts.jsx [hunk 2]
```diff
@@ -56,7 +62,7 @@ const HomePosts = (props) => {
           placeholder="Search Posts" 
           value={search} 
           onChange={e => handleSearch(e)}/>
-        <button>New Post</button>
+        <Link to="/createPost"><button>New Post</button></Link>
```
Comment (Can-Tina) [inline src/client/components/homeComponents/HomePosts.jsx:65]: Niceeee

File: src/client/styling/homePage.css [hunk 3]
```diff
@@ -58,14 +58,18 @@
...
   }
 
   .post {
       margin: 10px;
-      border: dotted 1px black;
+      border: solid 1px rgba(var(--clr-pokeblue), 0.3);
+      box-shadow: 1px 1px 5px rgba(0, 0, 0, 0.301);
+      border-radius: 3px;
       padding: 10px;
+      background-color: rgba(250, 235, 215, 0.74);
```
Comment (Can-Tina) [inline src/client/styling/homePage.css:72]: Looks much better

File: src/client/components/homeComponents/HomePosts.jsx (hunk 1 above)
Comment (vherus) [inline src/client/components/homeComponents/HomePosts.jsx:54]: I'm guessing this is here for testing, best to remove this before merging the PR

**Review by vherus — APPROVED** [review]

**Pull request merged**